.travis.yml

```diff
@@ -33,13 +35,15 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
 
+
+
 matrix:
   fast_finish: true
   include:
     - os : linux
       env: NAME='minimal'
            PYTHON_VERSION=2.7
-           SETUP_CMD='--with-memleak'
+           MAIN_CMD='python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak && python ./testsuite/MDAnalysisTests/mda_nosetests utils --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak'
```

Comment:
I don't understand how I can use `SETUP_CMD`.

Okay, will split the tests list into variables as before.

I thought we wanted `&&` ??

> It makes a very long and ugly command, but there are at least three benefits there:
> 
> * All the test will run; with what you have now, if a mda_nose command fails the next ones do not run beause of the &&
https://github.com/MDAnalysis/mdanalysis/pull/1370#issuecomment-304439942